src/components/screens/monitor/delegates/delegates.js

```diff
@@ -37,12 +50,12 @@ const Delegates = ({
       header: t('Address'),
       /* eslint-disable-next-line react/display-name */
       getValue: ({ address }) => <AccountVisualWithAddress {...{ address, isMediumViewPort }} />,
-      className: (filters.tab === 'active'
+      className: (activeTab === 'active'
         ? [grid['col-xs-3'], grid['col-md-3']]
```

Comment:
Why we don't use string letrals instead? Cause they're meant for this purpose exactly.

Reply:
This goes for all other occurances of `Array.join` in this component. My concerns why should we define an array and then merge its members right away?

Reply:
fixed, I made the changes